package.json

```diff
@@ -98,7 +98,7 @@
     "@hot-loader/react-dom": "^16.13",
```

Comment:
This is also happening in current odysee, where gif avatars being squished. 
![image](https://user-images.githubusercontent.com/64950861/148082453-ebaff503-d8a5-4fc0-971a-07b978be623c.png)

Reply:
Checked it on 2 channels and I'm unable to reproduce this. What's the name of this channel and where exactly does this happen?

Reply:
Happens in dropdown and profile (upper-right). 
Try setting your test channel thumbnail to https://cdn.dribbble.com/users/553939/screenshots/1558779/1024-dribbble.gif or https://d2r55xnwy6nx47.cloudfront.net/uploads/2016/07/RandomShape_Lead04.gif

Reply:
![image](https://user-images.githubusercontent.com/64950861/148103388-3aa849bd-7e5d-45c9-b9c8-c88016b1ebbd.png)

Reply:
This is low priority, by the way.  Just thought of highlighting in case the fix is simple.

Reply:
Fixed. At least in the two spots that you've mentioned. I wasn't able to find the issue anywhere else.
![image](https://user-images.githubusercontent.com/94839278/148200102-ab63d439-4356-4091-9c4e-1cffdfec3979.png)